conf/modules.config

```diff
@@ -32,6 +32,9 @@ params {
             args = '-M -K 100000000'
             publish_files = false
         }
+        'deepvariant' {
+            args = '--model_type=WGS'
```

Comment:
Would be nice to enable WES sequencing as well. If Sequencing type (WGS or WES) was a property of the analysis we could parse that variable and dynamically decide what model type to use in the deepvariant subworkflow.

Reply:
Could be something for the next PR also

Reply:
🤔  Yup! That's a good idea, we can add a global parameter `analysis_type=[wgs,wes]` to have as a dynamic variable. I'll add this in a coming PR (after glnexus has been merged) given that I'll need to research how/what part of the pipeline needs to be changed if analysis type is either of the two.

Reply:
For now, if a user wants to change `--model_type` they will need to make a `modules.config` that looks like the below and pass it in the CLI with the option `-c path/to/user.config`. It will be inherited and overwrite the parent args of deepvariant: 

```
params {
    modules {
        'deepvariant' {
            args = '--model_type=WES'
        }
    }
}
```
This won't change the amount of resources consumed and so probably another config needs to be written to accommodate exome size analyses. NEXT PR 😄 🚀

Reply:
Onwards and upwards!